packages/website/pages/api/user.js

```diff
@@ -0,0 +1,22 @@
+import mailchimp from '@mailchimp/mailchimp_marketing'
+
+const LIST_ID = '64f6e3fd11'
+
+/**
+ * Get user subscription info
+ * @param {import('next').NextApiRequest} req
+ * @param {import('next').NextApiResponse} res
+ */
+export default async (req, res) => {
+  mailchimp.setConfig({
```

Comment:
where is this running? cf pages does not support this

Reply:
next api routes have been removed